rails-snow-queen/app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,69 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ */
+$theme-colors: (
+  "primary":#61D5DD,
+  "warning":#fa7f28,
+  "success":#76B041,
+  "info": #99C5E6,
+  "secondary": #FFC914,
+  "light": #F6F7F7,
+  "dark": #177E89
+);
+
+@import 'bootstrap';
+
+
+header {
+  background-color: rgba(97, 213, 221, 0.452);
+  height: 20vh;
+}
+
+#map {
+  width: 100%;
+  height: 80vh;
+  background-color: grey;
+}
+
+#panel {
+  background-color: #177E89;
+}
+
+.map-btn {
+  border: 2px solid #fff;
+  background-color: #fff;
+  border-radius: 3px;
+  box-shadow: 0 2px 6px rgba(0,0,0,.3);
+  cursor: pointer;
+  margin-top: 5px;
+  text-align: center;
+  color: rgb(25,25,25);
+  font-family: Roboto,Arial,sans-serif;
+  font-size: 16px;
+  line-height: 20px;
+  padding-left: 5px;
+  padding-right: 5px;
+}
+
+.hidden {
+  display: none;
+}
+
+#displayAddress {
+  font-weight: bold;
+  text-shadow: 2px 2px 10px white;
+}
+
+#displayAddress > div:nth-child(1) {
```

Comment:
You're using SASS, so you can (should?) nest this:
```scss
#displayAddress {
  > div:nth-child(1) {
    margin-top: 30px;
  }
}
```
Also, you can use `:first-child` instead of `nth-child(1)` (bit easier on the eyes IMO)

Reply:
Didn't know I could do that, still learning the magic of SASS

Reply:
By the way, what I actually meant was:
```sass
#displayAddress {
  font-weight: bold;
  text-shadow: 2px 2px 10px white;

  > div:first-child {
    margin-top: 30px;
  }
}
```

Reply:
Done!